riscv_config/constants.py

```diff
@@ -6,6 +6,18 @@
 isa_schema = os.path.join(root, 'schemas/schema_isa.yaml')
 debug_schema = os.path.join(root, 'schemas/schema_debug.yaml')
 platform_schema = os.path.join(root, 'schemas/schema_platform.yaml')
+Zvl_extensions = [
+        "Zvl32b", "Zvl64b", "Zvl128b", "Zvl256b", "Zve512b", "Zvl1024b"
+]
+Zvef_extensions = [
+        "Zve32f", "Zve64f"
+]
+Zved_extensions = [
+        "Zve64d"
+]
+Zve_extensions = [
+        "Zve32x",  "Zve64x"
+] + Zvef_extensions + Zved_extensions
```

Comment:
I like this way of doing things, we should probably consider breaking up the existing `Z_extensions` list in a similar way - at least for the Zp and Zb lists that were added here: https://github.com/riscv-software-src/riscv-config/pull/118#discussion_r1119355158